src/main/java/io/jenkins/plugins/opentelemetry/job/MonitoringAction.java

```diff
@@ -158,6 +163,13 @@ public List<ObservabilityBackendLink> getLinks() {
         ).collect(Collectors.toList());
     }
 
+    private static boolean isPluginConfigured(JenkinsOpenTelemetryPluginConfiguration pluginConfiguration) {
+        return pluginConfiguration
+            .toOpenTelemetryConfiguration()
+            .toOpenTelemetryProperties()
+            .get("otel.exporter.otlp.endpoint") == null;
```

Comment:
I think this is needed rather than just checking the URL are you can configure the plugin through properties and environment variables too from that I can tell. I think this check covers everything though.

Its only triggered when no visualisation backend is setup anyway